lib/helper/Polly.js

```diff
@@ -134,8 +160,20 @@ class Polly extends Helper {
     return handler.intercept((_, res) => res.send(data));
   }
 
+  _mockRequestForWebDriver(method, oneOrMoreUrls, dataOrStatusCode, additionalData) {
+    const webDriverIOConfigUrl = this.helpers.WebDriver && this.helpers.WebDriver.options.url;
+    this.browser.execute(
+      pollyWebDriver.mockRequest,
+      method,
+      oneOrMoreUrls,
+      dataOrStatusCode,
+      additionalData || undefined,
+      this.options.url || webDriverIOConfigUrl,
+    );
+  }
+
   _checkIfMockingStarted() {
-    return this.polly && this.polly.server;
+    return (this.polly && this.polly.server) || this.browser;
```

Comment:
sorry, I didn't understand why `this.browser` should indicate that mocking started

Reply:
Because I am creating am storing the browser object when I start mocking with web Driver, so I thought I would use that to see if mocking started, as I don't have access to the Polly object. I see that it isn't very obvious to understand.

Reply:
> So I changed it to check and return if polly object has been created in browser context.